cdap-ui/app/directives/epoch/charts.js

```diff
@@ -94,7 +101,7 @@ ngEpoch.controller('epochController', function ($scope, $compile, caskWindowMana
 ngEpoch.directive('epochPie', function () {
   return angular.extend({
     link: function (scope, elem, attr) {
-      scope.initEpoch(elem, 'pie', attr);
+      scope.initEpoch(elem, 'pie', attr, {updateFull: true});
```

Comment:
For pie chart, each time we get new data, we update the entire chart (no notion of sending in a stream).